package.json

```diff
@@ -70,7 +70,7 @@
     "vuelidate": "^0.7.4",
     "vuex": "^3.5.1",
     "vuex-connect": "^2.0.1",
-    "vuex-persistedstate": "^3.1.0",
+    "vuex-persistedstate": "^4.0.0-beta.1",
```

Comment:
we need to pin this to v3 until we migrated to vue 3/vuex 4.

Reply:
It's not clear to me quite how to pin it, I tried:
- `"^3.1.0"`
- `"3.1.0"`
- putting "vuex-persistedstate": "3.0.0" into `"resolutions"` (https://classic.yarnpkg.com/en/docs/selective-version-resolutions/#toc-how-to-use-it)

... but in all cases doing `yarn upgrade --latest` gives us v4. Hmm. Am I missing something?